hat/assets/js/apps/Iaso/domains/mappings/components/CreateMappingVersionDialogComponent.js

```diff
@@ -60,7 +48,7 @@ const CreateMappingVersionDialogComponent = ({
             payload.program = dataset;
         }
 
-        createMappingRequest(payload).then(() => {
+        createMappingRequest.mutate(payload).then(() => {
```

Comment:
you need to use mutate async to have a promise, or pass a onsucces function in useCreateMappingMutation options to trigger the onclose when saved